draft-ietf-acme-acme.md

```diff
@@ -187,12 +187,11 @@ which the client can present to a human user.
 ~~~~~~~~~~
 
 
-Once the client is registered, there are three major steps it needs to take to
+Once the client is registered, there are two major steps it needs to take to
 get a certificate:
 
 1. Apply for a certificate to be issued
 2. Fulfill the server's requirements for issuance
-3. Finalize the application and request issuance
```

Comment:
Maybe change to "Await issuance and download the issued certificate" instead of deleting